shephard-core/src/main/java/com/devsda/platform/shepherdcore/service/NodeExecutor.java

```diff
@@ -74,7 +74,6 @@ public NodeResponse call() throws Exception {
             Document executionDetailsDoc = executionDocumentService.fetchExecutionDetails(this.node.getObjectId(),this.node.getExecutionId());
             try {
                 String executionDataJson= ((Document)executionDetailsDoc.get("executionData")).toJson();
```

Comment:
Why this again ?

Reply:
executionData field is in Mongo Document.